### PR TITLE
haaga-helia-university-of-applied-sciences-harvard.csl: no date cites need to be unique

### PR DESCRIPTION
**Pull request by mrautio-p**
Make cites unique when citing references from the same author without known issuing date.
Example of unique citing of four different references: (Musk s.a.a; Musk s.a.b; Github s.a.; Musk 2006; Musk 2007a; Musk 2007b)

### CODE REVIEW

File: haaga-helia-university-of-applied-sciences-harvard.csl [hunk 1]
```diff
@@ -167,6 +167,8 @@
       </else-if>
       <else>
         <text term="no date"/>
+        <!-- no date cites need to be unique -->
+        <text variable="year-suffix" prefix=" "/>
```
Comment (adam3smith) [inline haaga-helia-university-of-applied-sciences-harvard.csl:171]: If you add year suffix explicitly in one place, you also have to explicitly add it to the actual date.

**Review by adam3smith — CHANGES_REQUESTED** [review]

File: haaga-helia-university-of-applied-sciences-harvard.csl (hunk 1 above)
Comment (mrautio-p) [reply haaga-helia-university-of-applied-sciences-harvard.csl]: removed the attribute. the prefix attribute was incorrect and shouldn't be there in the first place. thanks for noticing it!

**Comment by mrautio-p** [comment]
@adam3smith made the requested change. the prefix attribute was incorrect

**Comment by adam3smith** [comment]
Sorry you misunderstood my comment: once you set year-suffix explicitly on the no date term, you also need it on the year. Otherwise you won't get 2001a, 2001b anymore

**Pull request merged**